tests/micro/zephyr/test_ms_tuning.py

```diff
@@ -95,7 +95,10 @@ def test_ms_tuning_conv2d(workspace_dir, board, microtvm_debug, use_fvp, serial_
     builder = get_local_builder_micro()
     with ms.Profiler() as profiler:
         with get_rpc_runner_micro(
-            platform=platform, options=project_options, session_timeout_sec=120
+            platform=platform,
+            options=project_options,
+            session_timeout_sec=120,
+            serial_numbers=["0", "1"],
```

Comment:
To be able to run this test in CI with physical devices, we need to pass the serial numbers. I suggest to reuse the `serial_number` pytest argument. We could use this argument in the format of `SERIAL_1,SERIAL_2,...,SERIAL_N` when we want to tune with N number of devices and parse the string in `serial_number` pytest argument function and return a list of serial numbers. Then we could pass the serial numbers to this function.
Then in this function you could check if serial_number arg is an empty list, just set it to `["0", "1"]`.

Reply:
Thanks @mehrdadh, made the change to use the serial number when available. PTAL.